ctapipe/io/tableloader.py

```diff
@@ -8,7 +8,7 @@
 from ..instrument import SubarrayDescription
 from .astropy_helpers import read_table
 
-__all__ = ["get_structure", "TableLoader"]
+__all__ = ["get_tel_ids", "get_structure", "TableLoader"]
```

Comment:
If we  make this a public method (as opposed to just an internal function) I think this should be a method of `SubarrayDescription`.

Reply:
Agreed - I put it here now to work on the unit tests, but we can move it later in any case

Reply:
there is already a `SubarrayDescripton.get_tel_ids_for_type()`, why not use that?

Reply:
that gives all the tel_ids, while this function extracts them from any subset defined by something like `Union[List[int], List[TelescopeDescription]]`

Reply:
so we can always get them in case someone does e.g. read_events([1,2,"MST_MST_NectarCam"]) or any other combination

Reply:
Ah, I see it's more general.